utils/plotting.py

```diff
@@ -78,10 +78,17 @@ def plot_curves(df, class_to_info):
             sample_weight_map[class_label] = len(y_true)/np.sum(y_true == class_label)
         sample_weight = y_true.map(sample_weight_map)
 
-        fpr, tpr, _ = roc_curve(y_true, y_score, sample_weight=sample_weight)
-        auc_score = roc_auc_score(y_true, y_score, sample_weight=sample_weight)
-        precision, recall, _ = precision_recall_curve(y_true, y_score, sample_weight=sample_weight)
-        ap_score = average_precision_score(y_true, y_score, sample_weight=sample_weight)
+        fpr, tpr, _ = roc_curve(y_true, y_score)#, sample_weight=sample_weight)
+        auc_score = roc_auc_score(y_true, y_score)#, sample_weight=sample_weight)
+        precision, recall, _ = precision_recall_curve(y_true, y_score)#, sample_weight=sample_weight)
+        ap_score = average_precision_score(y_true, y_score)#, sample_weight=sample_weight)
+
+        # Storing the curves for comparision, note very very very slow for running
```

Comment:
Sorry for the late reply, I was sort of on vacation with my family. 

The reason for logging them was that I could directly compare the PR and ROC curves in the mlflow UI rather than finding the curves in a separate notebook as I did. The implementation idea was from this issue: ([https://github.com/mlflow/mlflow/issues/2451](https://github.com/mlflow/mlflow/issues/2451)). 

But this slowed down the evaluation considerably, and so had commented it out for now.

Reply:
Sure, hope you've had a nice time on holidays!🙂 

Ah, I see, indeed that would be good but unfortunately `mlflow` doesn't support this option. The workaround you shared is a neat one, but it is slow probably because it has to log O(1000) values. Therefore, I propose to remove these commented lines.